dcos_launch/config.py

```diff
@@ -121,36 +123,45 @@ def get_validated_config(user_config: dict, config_dir: str) -> dict:
     # use the intermediate provider-validated config to add the platform schema
     platform = validator.normalized(user_config)['platform']
     if platform == 'aws':
-        validator.schema.update({
-            'aws_region': {
-                'type': 'string',
-                'required': True,
-                'default_setter':
-                    lambda doc:
-                        os.environ['AWS_REGION'] if 'AWS_REGION' in os.environ
-                        else util.set_from_env('AWS_DEFAULT_REGION')},
-            'disable_rollback': {
-                'type': 'boolean',
-                'required': False,
-                'default': False}})
-        if provider == 'onprem':
-            validator.schema.update(AWS_ONPREM_SCHEMA)
+        if provider == 'onprem-terraform':
```

Comment:
I think this could probably just be `terraform` as what happens under the hood is not our concern.

Reply:
Done